test/conformance/testdata/conformance.txt

```diff
@@ -112,15 +112,15 @@ test/e2e/common/projected.go: "should be consumable from pods in volume as non-r
 test/e2e/common/projected.go: "should be consumable from pods in volume with mappings"
 test/e2e/common/projected.go: "should be consumable from pods in volume with mappings and Item Mode set"
 test/e2e/common/projected.go: "should be consumable in multiple volumes in a pod"
-test/e2e/common/projected.go: "optional updates should be reflected in volume"
+test/e2e/common/projected.go: "updates should be reflected in volume"
 test/e2e/common/projected.go: "should be consumable from pods in volume"
 test/e2e/common/projected.go: "should be consumable from pods in volume with defaultMode set"
```

Comment:
this one as well (see line 110 currently)

Reply:
Ok after splitting them into separate files, I am still seeing some duplicates.  So far it looks like at least one of the duplicate tests is a superset of the other.  Can I remove conformance tests?

Reply:
I'm not sure what the rules for removing conformance tests are, They're expected to not change post-release but I'm not sure about during a release.

@spiffxp might be able to comment on this.

Reply:
er and yeah splitting the files wouldn't be enough I think, they'd need split-file-matching contexts I think? the generated `[sig-storage] Projected should be consumable from pods in volume with mappings [NodeConformance] [Conformance]` entry needs to be unique

Reply:
I changed the top level Describe() of each new file to be unique

Reply:
maybe we need to update the walker that generates this list, it strips out a lot